### ISSUE TITLE
Unless volume keys are mapped, use media volume.

### ISSUE THREAD

**Issue by easyaspi314** [mupen64plus-ae/mupen64plus-ae#455]
Preferably, we should have the volume key control media volume ingame (unless it is volume key controled).
        
 Call this in GameActivity.java and GameActivityXperiaPlay.java:

```java
@Override
public void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    // If audio is enabled, and we don't have volume keys mapped, we should force the volume buttons to control media volume 
    SharedPreferences mPreferences = PreferenceManager.getDefaultSharedPreferences(this);
    boolean volKeyMapped = mPreferences.getBoolean("inputVolumeMappable", false);
    GlobalPrefs mGlobalPrefs = new GlobalPrefs(this);
    if (!volKeyMapped && mGlobalPrefs.audioPlugin.enabled) {
     setVolumeControlStream(AudioManager.STREAM_MUSIC);
    }
    // rest of onCreate. We call this early so the user can change volume in the start process. 
}
```

**Comment by fzurita** [comment]
Hmm.... in my devices, while in Game, the volume keys already seem to control the "media volume".

**Comment by easyaspi314** [comment]
Are you using sles?

**Comment by fzurita** [comment]
Yep

**Comment by easyaspi314** [comment]
I have two JB 4.1.2 devices and both don't control volume in sles properly.
(Samsung Galaxy S II Skyrocket, LG Optimus Dynamic II). It may just be that
specific version, but I am not sure because all of my devices run 4.1.
I think we should do it anyways, to be safe.

Well unless something changed to fix it since b2be8ff I don't know. Can
someone build an APK for me? NDK doesn't work on my laptop for some reason.

**Comment by fzurita** [comment]
I can make that change and do a test build.

**Comment by easyaspi314** [comment]
Thanks, fzurita!

**Comment by fzurita** [comment]
Ok, I made a build, try this:

https://drive.google.com/file/d/0B57Ioy26LWegVXNHRWZMNEJXQzg/view?usp=sharing

**Comment by easyaspi314** [comment]
It works perfectly! Thanks.

**Comment by easyaspi314** [comment]
Fixed in #470.
